source/dbo/Stored Procedures/Data Migration/uspGenerateMigrationResultsData.sql

```diff
@@ -231,23 +231,6 @@ BEGIN TRY
 	FROM  [dbo].[MigrationRunResults] mrr
 		INNER JOIN AggregatedResults ag ON ag.LegacyImportMigrationRunId = mrr.MigrationRunId AND ag.OldNotificationId = mrr.MigrationNotificationId
 
-
-		
-	--and finally there may be a few rows where the record did not migrate because of an error on another record in its group
-	--so it has no error records itself
-
-	
-	UPDATE mrr
-		SET mrr.MigrationResult = 'Error',
-		mrr.MigrationNotes = li.[Notes]
-	FROM  [dbo].[MigrationRunResults] mrr
-		INNER JOIN 
-			[$(NTBS)].[dbo].[LegacyImportNotificationOutcome] li ON li.LegacyImportMigrationRunId = mrr.MigrationRunId AND li.OldNotificationId = mrr.MigrationNotificationId
-	WHERE mrr.MigrationRunId = @MigrationRunID
-	AND mrr.MigrationResult IS NULL
```

Comment:
The script was previously making the assumption that for these records the `MigrationResult` has not yet been set - but this is not true - it was being set in the update above.